tests/README.rst

```diff
@@ -57,14 +57,17 @@ the command::
 
 To prepare testing mock environment call::
 
-    ./scripts/testing/setup-mock-test-env.py [mock-configuration]
+    ./scripts/testing/setup-mock-test-env.py --init [mock-configuration]
 
 Mock configuration can be path to a file or name of file in `/etc/mock/*.cfg`
 without suffix. For detail configuration look on the script help output.
 
 Then you can run tests by::
 
-    mock -r [mock_configuration] --chroot -- "cd /anaconda && ./autogen.sh && ./configure && make ci"
+    ./scripts/testing/setup-mock-test-env.py -ut [mock-configuration]
```

Comment:
Note: There's `--copy` and `--update`, but it seems `--update` works just fine for the first time as well. So for someone who isn't interested in the details, this seems to be a reasonable standard command?

Reply:
"need" in setup-mock-test-env.py, of course. Do you want me to document both in README, though? Or leave that to the "see --help" part?